frontend/src/index.test.js

```diff
@@ -1,37 +1,2524 @@
+            "concept_code": "1085171000119108",
+            "invalid_reason": null,
+            "domain_cnt": 1,
+            "domain": "condition_occurrence",
+            "total_cnt": 200052,
+            "distinct_person_cnt": "13051",
+            "isItem": true,
+            "status": "Def"
+        },
+        {
+            "concept_id": 46269848,
+            "concept_name": "Abscess of intestine co-occurrent and due to chronic ulcerative rectosigmoiditis",
+            "domain_id": "Condition",
+            "vocabulary_id": "SNOMED",
+            "concept_class_id": "Clinical Finding",
+            "standard_concept": "S",
+            "concept_code": "1085231000119100",
+            "invalid_reason": null,
+            "domain_cnt": 1,
+            "domain": "condition_occurrence",
+            "total_cnt": 800,
+            "distinct_person_cnt": "200",
+            "isItem": true,
+            "status": "Def"
+        },
+        {
+            "concept_id": 46269850,
+            "concept_name": "Fistula of large intestine due to chronic ulcerative rectosigmoiditis",
+            "domain_id": "Condition",
+            "vocabulary_id": "SNOMED",
+            "concept_class_id": "Clinical Finding",
+            "standard_concept": "S",
+            "concept_code": "1085251000119106",
+            "invalid_reason": null,
+            "domain_cnt": 1,
+            "domain": "condition_occurrence",
+            "total_cnt": 552,
+            "distinct_person_cnt": "106",
+            "isItem": true,
+            "status": "Def"
+        },
+        {
+            "concept_id": 46269851,
+            "concept_name": "Rectal hemorrhage due to chronic ulcerative rectosigmoiditis",
+            "domain_id": "Condition",
+            "vocabulary_id": "SNOMED",
+            "concept_class_id": "Clinical Finding",
+            "standard_concept": "S",
+            "concept_code": "1085271000119102",
+            "invalid_reason": null,
+            "domain_cnt": 1,
+            "domain": "condition_occurrence",
+            "total_cnt": 60946,
+            "distinct_person_cnt": "5194",
+            "isItem": true,
+            "status": "Def"
+        },
+        {
+            "concept_id": 46269951,
+            "concept_name": "Abscess of intestine co-occurrent and due to ulcerative colitis",
+            "domain_id": "Condition",
+            "vocabulary_id": "SNOMED",
+            "concept_class_id": "Clinical Finding",
+            "standard_concept": "S",
+            "concept_code": "1092841000119100",
+            "invalid_reason": null,
+            "domain_cnt": 1,
+            "domain": "condition_occurrence",
+            "total_cnt": 4696,
+            "distinct_person_cnt": "1505",
+            "isItem": true,
+            "status": "Def"
+        }
+    ]
+};
+
+// TESTS
+// TODO: What are some good assertions, given various inputs?
+// - makeGraph()
+test('test makeGraph() - diagram case', () => {
```

Comment:
## Test progress: `makeGraph()`
I don't know if these outputs are correct. That takes more work and possibly your help.
I also don't know if these are what I should be testing.

Cases:
- diagramCase

Assertions:
- `expect(graph.directedSize).toBe(32);`
- `expect(graph.nodes()).toStrictEqual(uniqueConcepts);`  - `uniqueConcepts` is what I calculated as unique list of concepts in the edges of diagram case